### DISCUSSION TITLE
杂志主页

### DISCUSSION THREAD

**Discussion by CreeperIsASpy** [Meloong-Git/PCL#4926] [自定义主页]
![image](https://github.com/user-attachments/assets/f00ffe51-d0a8-4eeb-85a1-b6b5b62b0f5f)

更新时间: 周一下午到周二下午(本人初中生学业紧张理解一下)
直链链接: 
`http://pclhomeplazaoss.lingyunawa.top:27000/d/Homepages/Ext1nguisher/Custom.xaml`(主页市场oss, @JingHai-Lingyun )
仓库地址: `https://gitee.com/planet_of_daniel/pcl2-homepage-fhg` (已迁徙出github, 原仓库不再使用。)
由 @Light-Beacon 的代码 ( 已遵守相关协议 ) 更改而来。
内容取自[中文 Minecraft Wiki](zh.minecraft.wiki) 中"本周页面"板块以及[MC官网](minecraft.net)。

____________________

使用时推荐的启动器窗口大小：908x1180（可在注册表编辑器中更改）

_____

可加群 `967337278` 交流或方便地提出意见。
当前加群密码：
```
cprisaspyMGHP2025
```

__________________

欢迎大家提出修改意见或建议!

**Comment by iPodtouch6557** [comment]
如果是木桶这种随机的玩意占版面的话，其实意义不大（当然应该会改的）
可以选择最新版本或者玩家普遍关注的物品，甚至是备受瞩目的bug等等（除了新闻）

**Reply by Joker2184** [reply]
（和新闻冲突了吧（
他现在是基于本周页面刷的
之前老的讨论有人推荐这个他就做了

**Reply by CreeperIsASpy** [reply]
我现在没空手动更新（
我之前也考虑过做更有意思的主页~~杂志主页~~
但是没空啊
实在抱歉

**Reply by Chiloven945** [reply]
嗯？这主页从 Wiki 取的，所以说对内容有异议应该找 Wiki？

**Reply by iPodtouch6557** [reply]
> 嗯？这主页从 Wiki 取的，所以说对内容有异议应该找 Wiki？

好吧……不知道这个玩意呢

**Comment by MoYuan-CN** [comment]
![image](https://github.com/user-attachments/assets/d2db0382-4953-44ca-943d-128b65466e14)
Icon 大了……

**Reply by MoYuan-CN** [reply]
![image](https://github.com/user-attachments/assets/a9b3a30f-45e9-4186-a404-37147d23265e)
两张 Card 看起来不是一个风格…… 有点难受

**Reply by CreeperIsASpy** [reply]
![image](https://github.com/user-attachments/assets/306460a9-680f-4271-92aa-829caf4fc7b3)
改图标之后视觉效果有没有好一点，或者还是大？
~~我眼瞎~~

**Reply by MoYuan-CN** [reply]
还是大…… 大抵是构建器处理的不好…… Icon 本身没问题……

**Reply by Joker2184** [reply]
> ![image](https://private-user-images.githubusercontent.com/162774918/376034656-306460a9-680f-4271-92aa-829caf4fc7b3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkzMDI2NDcsIm5iZiI6MTcyOTMwMjM0NywicGF0aCI6Ii8xNjI3NzQ5MTgvMzc2MDM0NjU2LTMwNjQ2MGE5LTY4MGYtNDI3MS05MmFhLTgyOWNhZjRmYzdiMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxOVQwMTQ1NDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zMjE1ZTgyNjNkMTkzYjY4MWE1MjdjZjI1Njk3ZmI3NDg4YTI2OWFiN2RiMzA2OGU3YTY3NjQyNDM5ZGU5OTZhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.gZTEQAD9IhztA4wVPyVJDNBWU9uSqBE0cW3TQ7BMxE8) 改图标之后视觉效果有没有好一点，或者还是大？ ~我眼瞎~

大.jpg 不好看.jpg

**Comment by LTCatt** [comment]
“最新版本” 卡片的上间距小了，和下间距不一致。
`Made By 最亮的信标` 容易被误解为这个主页的作者是信标，应该更换表述，例如使用了由最亮的信标制作的模板，之类的。
没有提供版本号。详见 https://github.com/Hex-Dragon/PCL2/discussions/2528 第二条。
不建议使用 `--` 作为间隔符，考虑换成 ` - `（空格，减号，空格）。

**Reply by CreeperIsASpy** [reply]
收到，光速改进中
另外，最新版本卡片是用的信标的api，不在我的管辖范围
@Light-Beacon 修一下？

**Reply by Light-Beacon** [reply]
可以自己改…

**Reply by CreeperIsASpy** [reply]
自己改，指设置上面卡片的下边距。。。好吧
![image](https://github.com/user-attachments/assets/c580b641-f9ea-4b58-a660-e69087c5aeb9)
另外，有人能help me with this  吗？
我代码行都是对的，没有语法错误。。我要疯了

**Reply by MoYuan-CN** [reply]
大概率是你的生成器不规范（缩进）输出导致的，我格式化完就好了，自己对着下面我格式化后的文档对着看看？
[Custom.xaml.txt](https://github.com/user-attachments/files/17449046/Custom.xaml.txt)

---
另外这里的按钮错误的降了半em
![image](https://github.com/user-attachments/assets/ebea2d9c-c89b-4a2c-a04a-a443967a093a)

---
上边反馈的 Icon 记得去修
![image](https://github.com/user-attachments/assets/7edc2c3d-37f3-428f-96e6-b8d57bd42dde)

**Reply by CreeperIsASpy** [reply]
icon这个我没法修，因为我发现这个icon尺寸无论怎么改，大小都不变，给我红温了一下午。。。
> 另外这里的按钮错误的降了半em

懂，但是生成器代码又得长 100 个字符了 ~~（悩）~~

**Comment by CreeperIsASpy** [comment]
- [x] 针对下方"图为……"一行字中`<Underline>`控件的`Margin`做特殊处理（使用114行`if`语句实现）。
- [x] 将原本的`--`间隔符，换成` - `（空格，减号，空格）。
- [x] 将容易引起歧义的`Made By 最亮的信标`, 改为`使用了由最亮的信标制作的模板`之类的文字提示。
- [x] 修复“最新版本”卡片上间距与下间距不一致的问题。
- [x] 加入版本号文件，并在生成器中更新，同步到`当前版本`按钮中。
作者一直在努力，加油！

**Reply by CreeperIsASpy** [reply]
感谢WorldHim的修改orz

**Comment by Ad-closeNN** [comment]
![image](https://github.com/user-attachments/assets/9fc35114-950e-4faa-a014-58def8414e49)
嗯...这个“本周页面--”可以去掉吧，只留下“诡异菌钓竿”（毕竟上面都说了本周页面，并且框内文字都没显示完整...）
或者可以拉长这个主题颜色框

---

另外这两个在我这看不清...
![image](https://github.com/user-attachments/assets/f66377d5-88c8-4098-83a2-294681d615bb)

使用链接：[https://gitee.com/planet_of_daniel/pcl2-homepage-fhg/raw/main/Custom.xaml](https://gitee.com/planet_of_daniel/pcl2-homepage-fhg/raw/main/Custom.xaml)

**Comment by MoYuan-CN** [comment]
![image](https://github.com/user-attachments/assets/008897d2-d16e-41c8-a164-d131812f7317)
压缩咯

**Comment by Joker2184** [comment]
转个话

![f2d6967471300d1797fab4ab6b1cbee7](https://github.com/user-attachments/assets/326fa24f-8190-4024-9886-d7fef0a0ccca)
主页中由龙猫以及别人提出的bug，除图标bug外(原因见上)
其他已经全部检修。
目前主页正常更新了两星期，且生成器可以正常工作。
之后准备改造加入官方杂谈板块，翻译一两篇"识海漫谈"内容加进去试试看。
最近作者的梯子坏了，因此无法直接github告知，望体谅。
感谢大家对主页的关注与支持!

**Reply by MoYuan-CN** [reply]
然后那句话和图片放在一起就非常割裂了（）
![image](https://github.com/user-attachments/assets/3f72afe4-6485-41bf-9432-7402dc0c337b)

**Comment by LTCatt** [comment]
#5012，目前不再允许使用 Gitee 作为内容提供地址，请更换一个地址 orz

**Comment by Joker2184** [comment]
作者....太忙（润了（

![QQ截图20241107163413](https://github.com/user-attachments/assets/7b1d6a17-43f3-4de2-854f-a0c430a0192a)

**Comment by CreeperIsASpy** [comment]
说明一下情况
正常更新(~~当然中间突然看见主页更新了某些奇奇怪怪的东西也是有可能的~~)
太忙了awa, 虽说期中考已经过去了，但是有点来不及，还有好多奇奇怪怪的bug待修......，所以更新如果晚了别急。
[主页](http://pclhomeplazaoss.lingyunawa.top:27000/d/Homepages/Ext1nguisher/Custom.xaml)挂到主页市场oss了，在这里先感谢一下 @JingHai-Lingyun 提供的良心服务! 
有空修bug，在这先挂个checklist
- [x] docker动态更新(可能根本不会搞)
- [x] 鸣谢卡片字体间距调整
- [x] 将各卡片中对主页的称呼全面从`中文 Minecraft Wiki 摘录主页` 改为 `杂志主页`
- [x] 加入翻译好的 mc 官网文章
- [x] 把下面有点看不清的`Author:CreeperIsASpy`图标换为文字并配上主页更新时间, 注意事项等内容。

**Reply by MoYuan-CN** [reply]
### OSS 服务提供者友情提醒
你不应当直接点击上文中的 `主页` 按钮，不然你会得到一个 ⚠️ 警告 ⚠️ 页面。
请复制下方链接并在 PCL 中联网加载
```
https://pclhomeplazaoss.lingyunawa.top:27000/d/Homepages/Ext1nguisher/Custom.xaml
```

**Reply by CreeperIsASpy** [reply]
@LTCatt 再看看？目前正常更新!

**Reply by LTCatt** [reply]
ok，下个版本加进去

**Reply by CreeperIsASpy** [reply]
谢谢!!

**Comment by CreeperIsASpy** [comment]
各位, 给个 svg 资源站呗
才发现下面`打开 Gitee `按钮的图标用的竟然是 Github 的图标......

**Reply by MoYuan-CN** [reply]
[https://icones.js.org/](https://icones.js.org/)
我主用的是 Mdi

**Reply by CreeperIsASpy** [reply]
谢谢！太有用了！所有icon都有！

**Reply by MoYuan-CN** [reply]
我怎么记得我好些天前和你说过这个……

**Comment by CreeperIsASpy** [comment]
### 速览: 博文翻译板块在路上了
_我知道你很急, 但请你先别急!_ 

![image](https://github.com/user-attachments/assets/c8e1c7df-b0ac-4660-a902-5fbd2f55f4d2)
预计本周周末发布

博文是手动翻译的，群友和很多朋友都建议我用 AI 翻译，这边解释一下为什么不这么做。
我并不害怕麻烦，我更害怕的是主页的质量变低，一个用 AI 水出来的板块不会是各位想要的。
并且，在翻译的过程中，我愈来愈感受到 AI 翻译的局限性，且觉得有必要增强这方面的技能。

<details><summary>另外。。。</summary>
<p>

![image2](https://github.com/user-attachments/assets/49c7dedd-6312-46f4-9842-65b83f7f855e)
手动用笔在学校翻译的。。
累死我啦！

</p>
</details> 
祝各位使用愉快！

**Comment by CreeperIsASpy** [comment]
### 博文速报板块发布！大更新！
_热爱可抵岁月漫长！_

请加载主页并翻到`最新版本`卡片下方，你将看到一个崭新的板块：`Minecraft官方博文`！
~~这下确实是坐实了杂志主页的名号了。。。~~
<details><summary>图片预览 (很长,告警) </summary>
<p>

![image](https://github.com/user-attachments/assets/f555359d-1996-479c-a7a8-84a15842cd71)
![image](https://github.com/user-attachments/assets/b96da2b9-bbd7-4183-803b-8696edbb7d95)

</p></details>

目前已经更新到 oss ，使用愉快！
另：最近OSS有那么一点点不稳定，如果实在有问题可以到gitee仓库下载源文件本地加载。

**Comment by CreeperIsASpy** [comment]
checklist好了！动态更新ready！再次感谢 @JingHai-Lingyun 和 @Joker2184 的 OSS 服务!!!!!

**Comment by Joker2184** [comment]
![QQ20241119-174125](https://github.com/user-attachments/assets/d2ba13c1-fc68-41dc-afa8-2198c69d8d42)
推荐把最新版本往下移 移到识海漫谈下方.jpg

如果可以把两个卡片间隙加到15.jpg 

不然挺突兀的.jpg
Logo还可以再缩小一点 最好比你ID字号稍大一点....

今天MC的文章有点多....一天8篇乱七八糟的文章 你需要翻译的就有3篇了.....
你可以考虑做个单独的页面 类似信标那种历史版本页面...毕竟翻译完的东西还是有价值的对吧...

**Reply by Joker2184** [reply]
请替换此链接临时切换至其他图床链接....
OSS下载链接返回的图片是空图片.jpg
浏览器下载出来后缀是avif
需要测试问题....
```
<StackPanel>
<Image Style="{StaticResource InnerImage}" Source="http://pclhomeplazaoss.lingyunawa.top:26994/d/Homepages/Ext1nguisher/trailruins-ingame.jpg"/>
<TextBlock Text="游戏中的古迹废墟" Style="{StaticResource imgTitle}" />
</StackPanel>
```

**Reply by Joker2184** [reply]
![QQ20241119-181305](https://github.com/user-attachments/assets/7c09cc6c-79a1-4241-9cdb-61330bc1821f)

官网也是 后缀jpg 打开保存是avif.....

**Reply by 3gf8jv4dv** [reply]
这张图片确实是 AVIF 格式。奇怪的应该是为什么后缀名对不上。

MediaInfo:
```
General
Complete name                            : trailruins-ingame.avif
Format                                   : avif
Codec ID                                 : avif (avif/mif1/miaf)
File size                                : 27.2 KiB

Image
ID                                       : 1
Format                                   : AV1
Format/Info                              : AOMedia Video 1
Codec ID                                 : av01
Width                                    : 748 pixels
Height                                   : 421 pixels
Color space                              : YUV
Bit depth                                : 8 bits
Stream size                              : 26.9 KiB (99%)
Color range                              : Full
Color primaries                          : BT.709
Transfer characteristics                 : sRGB/sYCC
Matrix coefficients                      : BT.601
Codec configuration box                  : av1C
```

**Reply by CreeperIsASpy** [reply]
我实在没办法了
格式工厂不支持转换avif...~~什么傻逼文件格式~~
@LTCatt 考虑让PCL支持加载avif ~~虽然知道不太可能~~

另： @Joker2184 关于卡片布局的意见收到，可以制作历史板块。我测试了一下，是可以加载的，既然可以用就行了。

**Reply by LTCatt** [reply]
这个不太可能。你截个图存 png 呗……

**Reply by Joker2184** [reply]
avif是什么奇奇怪怪的格式.jpg
你实在不行qq截图（）
然后主页图片推荐图床.jpg OSS目前策略调整为仅支持WebP 后续凌云会开始限制.jpg

**Reply by Chiloven945** [reply]
> avif是什么奇奇怪怪的格式.jpg
> 你实在不行qq截图（）
> 然后主页图片推荐图床.jpg OSS目前策略调整为仅支持WebP 后续凌云会开始限制.jpg

[av1图像格式](https://zh.wikipedia.org/wiki/AVIF)，近几年才出来的，简单来说就是把av1编码给图片用

**Reply by CreeperIsASpy** [reply]
唉……谁能想到还能出来这样的问题。。

**Comment by CreeperIsASpy** [comment]
最近在憋个大的板块，是api水的。。

**Reply by CreeperIsASpy** [reply]
希望能在 https://github.com/Hex-Dragon/PCL2/discussions/4990 之前搞出来吧，不然就抄袭了。。

**Comment by Joker2184** [comment]
机房PCDN锁宽带了
OSS停止服务（暂时）请切换至其他托管服务托管

**Reply by CreeperIsASpy** [reply]
> 另：最近OSS有那么一点点不稳定，如果实在有问题可以到gitee仓库下载源文件本地加载。

完了，成真了

**Comment by CreeperIsASpy** [comment]
一周最期待的时刻到来啦~！
OSS 服务正常，历史板块加入！
我们的第一篇历史博文是———古迹废墟！

**Reply by CreeperIsASpy** [reply]
@LTCatt 呃呃，下个正式版还要等几年啊……

**Reply by Joker2184** [reply]
> @LTCatt 呃呃，下个正式版还要等几年啊……

nnd bug一堆现在（

估计直接跳2.8.11了（

**Comment by LTCatt** [comment]
![image](https://github.com/user-attachments/assets/22b63b52-ad5d-4c56-a3a7-071d142dc61c)
卡片在折叠的时候……

**Reply by Joker2184** [reply]
~~哇哦~~

**Reply by LTCatt** [reply]
请在修复后回复本帖，不然这轮更新也上不了……

**Reply by CreeperIsASpy** [reply]
已修复，使用愉快。
~~别威胁，给我吓怕了，我还得上学呢……~~
@LTCatt

**Comment by LTCatt** [comment]
已添加至 2.8.11

**Comment by CreeperIsASpy** [comment]
已经采纳 @Joker2184 的建议，将最新卡片移动到最下方，且增加了历史板块，统一了页面 Margin 。
这边放个 checklist :
- [x] 迁徙图片至图床
- [x] 完善`reloader.py`~~不要三次运行两次崩.jpg~~
- [ ] `历史上的今天`板块 @shimoranla

另：下次反馈 bug 请到 gitee / github 仓库交 issue 而不是在这里水帖……~~放在仓库里我还可以当没看见~~

**Reply by LinQingYuu** [reply]
>历史上的今天板块 @shimoranla

百科的 API 包含了 html 标签，反正就很抽象，我也没啥好办法给去掉（

**Reply by CreeperIsASpy** [reply]
@shimoranla 我已经洗html了
到时候再推个PR发给你
笨办法，直接
```
import re 
while re.findall(r'<.*?>', 源字符串):
    源字符串 = re.sub(源字符串, r'<.*?>', '')
```
~~直接while delete~~

**Reply by LinQingYuu** [reply]
>到时候再推个PR发给你

不大懂正则表达式，有想法都可以发出来，大部分应该是能合并的

Edit:最近手上搓了三四个项目玩，不一定能顾上所有的内容，所以你可能会看到 PR 被搁置了一段时间才被合并，切勿着急

**Comment by CreeperIsASpy** [comment]
速报: OSS 又双叒叕炸了……

**Comment by Joker2184** [comment]
图形化炸了
换系统了
今天晚上搞好确保服务

**Comment by Joker2184** [comment]
@Ext1nguisher 已修复
链接不变 不需要重新上传文件 

~~为了你专门注册了一个gitee~~

**Reply by CreeperIsASpy** [reply]
6, 好的，收到。

> ~~为了你专门注册了一个gitee~~

呃，啥意思……

**Reply by Joker2184** [reply]
下载你的文件 上传（） 撒币gitee不能免登录下载

**Comment by CreeperIsASpy** [comment]
@LTCatt 请把完成之后在启动器内的署名改成CreeperIsASpy而不是Ext1nguisher
这是我的小号.png

**Comment by CreeperIsASpy** [comment]
@LTCatt 请把完成之后在启动器内的署名改成CreeperIsASpy而不是Ext1nguisher
这是我的小号.png

**Reply by Joker2184** [reply]
~~人家也是很忙的~~

**Reply by hejiehao** [reply]
你发了两次了

**Comment by LTCatt** [comment]
署名改了。
新版本已经整好了，但主页目前依然无法访问，如果今天还是恢复不了的话只能先删了把新版本更出去了……

**Reply by Joker2184** [reply]
端口改26994 ```http://pclhomeplazaoss.lingyunawa.top:26994/d/Homepages/Ext1nguisher/Custom.xaml```

我署名要不也改一下（->这个```ZeroPrism```

新闻速报新链接已正常```http://pclhomeplazaoss.lingyunawa.top:26994/d/Homepages/Joker2184/UpdateHomepage.xaml```

**Reply by Joker2184** [reply]
27000的防火墙估计凌云改炸了（

**Reply by LinQingYuu** [reply]
>如果今天还是恢复不了的话只能先删了把新版本更出去了……

龙猫可以考虑下 #5219，这样就不会拖慢更新进度了（

**Reply by Joker2184** [reply]
> > 如果今天还是恢复不了的话只能先删了把新版本更出去了……
> 
> 龙猫可以考虑下 #5219，这样就不会拖慢更新进度了（

我觉得可以（？ 到时候主页提供配置文件下载就行.jpg

**Reply by LinQingYuu** [reply]
>我不想拖慢更新进度

那就先删了吧，这个地址凌云不建议使用的（

**Reply by Joker2184** [reply]
> > 我不想拖慢更新进度
> 
> 那就先删了吧，这个地址凌云不建议使用的（

没事 到时候改一下就行 先用着吧

**Comment by CreeperIsASpy** [comment]
> 27000的防火墙估计凌云改炸了（

@LTCatt 这也不是我能修的啊……
这样，你先看看 [#5219](https://github.com/Hex-Dragon/PCL2/issues/5219) ,尽量先更着？……我不想拖慢更新进度

**Reply by CreeperIsASpy** [reply]
算了。
@LTCatt 用
```http://pclhomeplazaoss.lingyunawa.top:26994/d/Homepages/Ext1nguisher/Custom.xaml```
先更了吧，下个版本再改……
反正我主页也不大，应该还没人想黑……

**Reply by Joker2184** [reply]
> 欸，我这里用27000还能正常刷新主页……_呃呃，又好了？？_

缓存 ini未更新不会重新获取（

上面已经说了换26994（（（重复了

**Reply by MoYuan-CN** [reply]
参见
- https://github.com/HomePlaza-Of-PCL2/Discussions/discussions/6

中的通知

**Reply by LinQingYuu** [reply]
> 参见
> 
> * [服务状态面板 - Service State Dashboard HomePlaza-Of-PCL2/Discussions#6](https://github.com/orgs/HomePlaza-Of-PCL2/discussions/6)
> 
> 中的通知

要不然搭个 uptime kuma 得了

**Reply by Joker2184** [reply]
问题不大 现在WAF套在26994上 杂志主页端口也是26994安全性还是有保障的

**Comment by CreeperIsASpy** [comment]
### 紧急通知 ⚠

在24/12/15 19:45前，本主页所有图片都无法访问状态。
现已经发布紧急更新，将除`Wiki摘录板块`外所有图片迁徙至`hello图床`，修复完毕。

特此告知。使用愉快！

**Reply by Joker2184** [reply]
> ### 紧急通知 ⚠
> 在24/12/15 19:45前，本主页所有图片都无法访问状态。 现已经发布紧急更新，将除`Wiki摘录板块`外所有图片迁徙至`hello图床`，修复完毕。
> 
> 特此告知。使用愉快！

其实可以好像（

**Reply by LinQingYuu** [reply]
>其实可以好像（

凌云加了规则，请求图片现在应该是 403（

**Reply by MoYuan-CN** [reply]
现在所有的图片都会因为超频率被拦截的……

**Comment by CreeperIsASpy** [comment]
目前已知 Bug 列表：
- [x] `博文板块`中，`访问原址`按钮的链接地址与内容不对应
- [x] 下方`说明板块`中，`中文 Minecraft Wiki`和`MC 官网`的`Hyperlink`无法访问 （ @Ad-closeNN ）
- [x] `中文 Minecraft Wiki 板块`中，`WIKI`按钮的链接地址与内容不对应

**Reply by Joker2184** [reply]
> 目前已知 Bug 列表：
> 
> * [ ]  `博文板块`中，`访问原址`按钮的链接地址与内容不对应
> * [ ]  下方`说明板块`中，`中文 Minecraft Wiki`和`MC 官网`的`Hyperlink`无法访问 （ @Ad-closeNN ）
> * [ ]  `中文 Minecraft Wiki 板块`中，`WIKI`按钮的链接地址与内容不对应

hyper 直接换textbutton

**Reply by CreeperIsASpy** [reply]
已在`241223:f756cc5`中修复

**Comment by CreeperIsASpy** [comment]
# 新 UI 测试版
预览文件：[Custom.txt](https://github.com/user-attachments/files/18227946/Custom.txt)
> 呃……因为github不支持xaml, 所以是 `txt` 格式，下载的时候改一下拓展名就好啦

## 使用`TabControl`控件来改善挤成一堆的原版卡片样式。
### 优点：
- 方便查看，便于查找相关内容 ~~，防止眼瞎~~。
- 整体上，内容更符合"美观为上，实用为本，简洁为刚"的设计理念。

### 缺点：
- 方形边缘的直来直去有悖于 PCL2 的整体设计，影响观感。
- 上方的卡片似乎太小了，且难以将间距统一。

## 进行投票吧！
- 🚀 ： ✔️ 你认可新 UI 设计！
- 😕 ： ✖️ 你认为新 UI 设计还有有待改进之处！请在下方给出你的建议或意见！

**Reply by LinQingYuu** [reply]
![0](https://github.com/user-attachments/assets/b84a23a8-c002-425d-958a-d77b81acb835)

黑笔圈出的部分，图片加载过慢，有没有其他图床

红笔部分，个人感觉这个选项卡不好看（太老了），而且贴在上面感觉有点突兀，有没有圆润一点的

**Reply by CreeperIsASpy** [reply]
关于图床问题，我再声明一遍，**Wiki摘录板块的内容是绝对不会加图床的**
原因：
1. 图床是为加载速度太慢的图片源服务，然而 **中文** Minecraft Wiki 是国内服务，加载速度是稳定的。~~人话：你的问题~~
2. Wiki摘录板块更新频率很高，本来每次更新都是仓促进行的，大多数内容由`reloader.py`自动完成，然而上传图床并没有可供自动上传的命令行方法或指令，无法自动化。~~周中我还要写作业呢！~~

**Reply by Joker2184** [reply]
> 关于图床问题，我再声明一遍，**Wiki摘录板块的内容是绝对不会加图床的** 原因：
> 
> 1. 图床是为加载速度太慢的图片源服务，然而 **中文** Minecraft Wiki 是国内服务，加载速度是稳定的。~人话：你的问题~
> 2. Wiki摘录板块更新频率很高，本来每次更新都是仓促进行的，大多数内容由`reloader.py`自动完成，然而上传图床并没有可供自动上传的命令行方法或指令，无法自动化。~周中我还要写作业呢！~

无法自动化：为什么不问问万能的ChatGPT呢（

**Reply by CreeperIsASpy** [reply]
首先，Hello图床根本就没有可用的接口可以自动上传。
其次，我没梯子

**Reply by LinQingYuu** [reply]
>首先，Hello图床根本就没有可用的接口可以自动上传。

https://www.helloimg.com/page/api-docs.html

显然不是没有接口的说....

>人话：你的问题

应该算是突发性的加载缓慢，但如果硬要说网不行的话

~中国电信:你要被夹.jpg~

**Comment by CreeperIsASpy** [comment]
![e](https://github.com/user-attachments/assets/e89c9ad8-e6cd-439a-9d66-17222b274bb3)
`TabControl`的风格确实有点老了。
但是如图，分区分块改善观感这个是肯定得做的。
我现在在找办法提升观感吧，然后再不行就直接`打开帮助`按钮分成多个页面。
目前方式：[垃圾博客法](https://www.cnblogs.com/xiaomingg/p/8870825.html)

**Comment by CreeperIsASpy** [comment]
### 新 UI 改进测试完成! 
![image](https://github.com/user-attachments/assets/367e45a4-8003-45ca-b5ee-b3d537d350e9)
漂亮！预览文件：[Custom.txt](https://github.com/user-attachments/files/18281801/Custom.txt)
> 呃……因为github不支持`xaml`, 所以是`txt`格式，下载的时候改一下拓展名就好啦！

**Reply by Ad-closeNN** [reply]
![image](https://github.com/user-attachments/assets/b51d5ee9-fc3f-4edf-b4fd-dde57e26b5f1)
额，下划线？

**Reply by MoYuan-CN** [reply]
文本该飘得还是飘了
括号用中文的 `（）`

**Comment by Joker2184** [comment]
![QQ截图20250101102204](https://github.com/user-attachments/assets/6e70f78a-a4e8-41d1-9e0d-b7b3ab035cce)
在标准界面下可能会出现点小问题...

**Reply by CreeperIsASpy** [reply]
呃，不知道怎么搞自适应……

**Reply by MoYuan-CN** [reply]
猜你在找：Grid 布局

**Reply by CreeperIsASpy** [reply]
问题是，据我所知`TabItem`卡片每个都是单独的，没办法作为单独一个个对象布局。

**Comment by CreeperIsASpy** [comment]
### 新UI投入使用，版本号`250106:eb03ad9`

![image](https://github.com/user-attachments/assets/4d3c9550-0168-4b05-99fd-134f06d0d2bb)

使用`Grid`解决了最后的布局自适应问题。
进行了最后的小修小改，修复了文字链接无下划线的问题。
对一些板块的位置布局进行了大幅度调整。
祝
    使用快乐！

**Comment by CreeperIsASpy** [comment]
## To-do list：
- [ ] 修复所有 bug （见下方已知 bug 列表）。
- [ ] `历史上的今天`板块（加入后放在`其他`选项卡内。）
- [ ] 优化`reloader.py`脚本，实现`hello图床`API 自动上传图片。

## 目前已知 bug 列表：
- [x] 所有卡片组件上方上边距过大，导致一片莫名奇妙的空白区域。（[IBHDJY](https://gitee.com/planet_of_daniel/pcl2-homepage-fhg/issues/IBHDJY)）
- [ ] 标准宽度下选项卡文字压缩，按钮和文字重叠。（[IBI1OQ](https://gitee.com/planet_of_daniel/pcl2-homepage-fhg/issues/IBI1OQ)）
- [x] 中文 Minecraft Wiki 板块中，WIKI按钮的链接地址与内容不对应 ~~又忘了改了.png~~ （[IBHDKG](https://gitee.com/planet_of_daniel/pcl2-homepage-fhg/issues/IBHDKG)）

**Comment by CreeperIsASpy** [comment]
@LTCatt 问一句，这个自定义主页上预设了能不能领活跃橙啊……
<details><summary>一些小玩意</summary>
<p>

![image](https://github.com/user-attachments/assets/ee909c41-5b83-4651-91d3-1ebdf3ea2ce0)
很久了……
龙猫加油！支持~

</p>
</details>

**Reply by LTCatt** [reply]
可以领，丢个识别码？

**Reply by LinQingYuu** [reply]
> 可以领，丢个识别码？

活跃橙渠道 +1

**Reply by CreeperIsASpy** [reply]
> 可以领，丢个识别码？

活跃橙 AEBA-BB56-FA88-E93F @LTCatt

**Reply by CreeperIsASpy** [reply]
<details><summary>666</summary>
<p>

![image](https://github.com/user-attachments/assets/086160eb-99be-405f-afc4-f98f371ca864)

</p>
</details> 
第一次知道龙猫有午睡的习惯……
保重身体健康吧。

**Reply by CreeperIsASpy** [reply]
@LTCatt @LTCatt @LTCatt

**Reply by LinQingYuu** [reply]
> @LTCatt @LTCatt @LTCatt

理论上 @ 一次就行了（

**Reply by LTCatt** [reply]
你的解锁码：Qg0aKwxpQK1zlZ3oKyLxcTWLf/G5Ld93
在 更多 → 关于与鸣谢 → 输入解锁码 中使用，谢谢支持～

**Reply by CreeperIsASpy** [reply]
> 你的解锁码：Qg0aKwxpQK1zlZ3oKyLxcTWLf/G5Ld93 在 更多 → 关于与鸣谢 → 输入解锁码 中使用，谢谢支持～

🤩 谢谢！辛苦了~

**Comment by Joker2184** [comment]
??? 也没人跟我说能领活跃橙啊

**Reply by LTCatt** [reply]
新版本加描述里了 orz

**Reply by CreeperIsASpy** [reply]
因为我比较聪明.png

**Comment by CreeperIsASpy** [comment]
## 预告：除夕猪页会整个大活.png

**Comment by MacroMeng** [comment]
太好了qwq

[MacrosMeng - Outlook]
________________________________

**Comment by CreeperIsASpy** [comment]
最近时间全在打塞尔达，没啥时间搞翻译工作了。
最近有了{ladder}也可以访问互联网档案馆了……_要不把 MCBBS 的文章翻译拿过来，嘿嘿……_

**Comment by CreeperIsASpy** [comment]
## PCL2 杂志主页 `250123:b09c09f` 更新日志：
### 主要内容：
1. 博文板块选用了原 MCBBS 的文章。加入了以灰色小字书写的英文原文。重新加入了`访问原址`按钮。
2. 博文板块的作者栏目除了原来的译者栏，还新增加了译者栏目和原文发布日期说明。
3. 其他板块的"最新版本"卡片上方增加了用于提醒用户卡片内容可能不为最新的提示条。
4. 修复 Bug “`鸣谢`窗口的标题上方有冗余的空行”，@Ad-closeNN
5. 更新其他板块中"最新版本"卡片至`25w04a`
